starter_code/src/movies.js

```diff
@@ -1,19 +1,177 @@
 /* eslint no-restricted-globals: 'off' */
 
-// Iteration 1: All rates average - Get the average of all rates with 2 decimals 
+// Iteration 1: All rates average - Get the average of all rates with 2 decimals
+
+function ratesAverage(arr) {
+  const sum = arr.reduce(function(accumulator, currentValue) {
+    let number = parseFloat(currentValue.rate, 10);
```

Comment:
Here I don't understand the second parameter in your parseFloat. there is no second parameter. What was the logic here? :)